### PR TITLE
Remote batch insert

### PR DESCRIPTION
**Pull request by soujiro32167**
This odyssey started when I wanted to run a benchmark against OrientDB 3 in a Kubernetes cluster. A few days (and exceptions) later, I created this PR agains the benchmark repo: https://github.com/orientechnologies/graphdb-benchmarks/pull/1, seeking help.

After digging through the code, I realized that the deprecated `ODatabaseDocumentTx` could be converted to use the `OrientDB` API - and that's what I did here.

TL;DR:  `OGraphBatchInsertBasic` wasn't working with remote databases, this PR fixes and tests this

### CODE REVIEW

**Comment by luigidellaquila** [comment]
Merged, thanks!

Luigi

**Comment by soujiro32167** [comment]
Thanks, @luigidellaquila. BTW, I tried to sign the contributor license agreement and got an error: https://cl.ly/3Z2y2g142V0s

**Comment by luigidellaquila** [comment]
@lvca any idea?

**Comment by luigidellaquila** [comment]
@soujiro32167 could you please try again? it seems to work fine now

Thanks

Luigi

**Comment by soujiro32167** [comment]
I see the CLA as signed now! Thanks guys =]

**Comment by luigidellaquila** [comment]
Great, thanks!
